app/__main__.py

```diff
@@ -79,10 +84,17 @@ def is_stop_condition_met(results: Iterable[SubproblemResult]) -> bool:
     return any(r is not None for r in results)
 
 
-def create_broker(connection_settings: ConnectionSettings) -> Broker:
+def create_broker(connection_settings: ConnectionSettings, mapper: CommandMapper) -> Broker:
     logger = get_logger('broker')
     connection = NetworkConnection(connection_settings)
-    return LoggingBroker(connection, logger)
+    return LoggingBroker(connection, logger, mapper)
+
+
+def create_command_mapper() -> CommandMapper:
+    return CommandMapper() \
+        .register(SubproblemResultCommand) \
+        .register(ImAliveCommand) \
+        .register(NetTopologyCommand) 
```

Comment:
Move these back to the broker. In `__main__.py`, only domain commands should be registered, and network commands should be registered inside broker.